fsw/cfe-core/unit-test/sb_UT.c

```diff
@@ -6016,7 +6002,7 @@ void Test_RcvMsg_UnsubResubPath(void)
     SETUP(CFE_SB_Subscribe(MsgId, PipeId));
     ASSERT(CFE_SB_RcvMsg(&PtrToMsg, PipeId, CFE_SB_PEND_FOREVER));
 
-    ASSERT_TRUE(PtrToMsg == NULL);
```

Comment:
So this is curious, I need to confirm that the behavior has changed...But I believe the new behavior is correct (this is probably a side-effect of one of the other merges).